test/parallel/test-http-unix-socket-keep-alive.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+
+const server = http.createServer((req, res) => res.end());
+
+common.refreshTmpDir();
+
+server.listen(common.PIPE, () =>
```

Comment:
Any chance you can use port `0` and still have this test work? If not, it should go in `test/sequential` rather than `test/parallel`. (Otherwise, some other test using port `0` in parallel might conflict with `common.PORT`.)